src/services/kill-process-id.service.test.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+jest.genMockFromModule('child_process');
+jest.mock('child_process');
+
+const createOsMock = osName => {
+  jest.doMock('os', () => ({
+    homedir: jest.fn(),
+    platform: () => osName,
+  }));
+};
+
+const createPsTreeMock = (id, err, children) => {
+  jest.doMock('ps-tree', () => jest.fn((doomedId, cb) => cb(err, children)));
+};
+
+describe('killProcessId', () => {
+  const id = '12345';
+  beforeEach(() => {
+    jest.resetModules(); // reset os mock
+  });
+
+  it('should spawn taskkill on Windows', async () => {
+    createOsMock('win32');
+
+    const killProcessId = await import('./kill-process-id.service');
+    const childProcess = await import('child_process');
```

Comment:
Why dynamic `import()` here instead of just importing once at the top of the test file?

Reply:
I don't fully understand why this works and importing top-level doesn't, but it does work, so I'll just leave it.